internal/jet/sql_builder.go

```diff
@@ -253,6 +255,19 @@ func argToString(value interface{}) string {
 	}
 }
 
+func stringArrayQuote(val []string) string {
```

Comment:
Maybe we can use `pq.StringArray{}.Value()` instead, and cast first returned value to string.

Reply:
Like this?
```
func stringArrayQuote(val []string) string {
	// We'll rely on the internals of pq2.StringArray here. We know it will never return an error, and the returned
	// value is a string
	dv, _ := pq2.StringArray(val).Value()
	return dv.(string)
}
```

Reply:
Yeah, exactly.

Reply:
Done